Source/Fuse.Navigation/Router.ScriptClass.uno

```diff
@@ -218,6 +222,27 @@ namespace Fuse.Navigation
 			{
 				return _context.Wrap(value) as Node;
 			}
+
+			public void DiscardContext()
+			{
+				_context = null;
+			}
+		}
+
+		class MakeRequestClosure
```

Comment:
I wonder if RouterRequest could also just have an optional  `Router` and then use `MakeRequest` as the action. It'd save an allocation and remove the need for this class.

Reply:
That sounds like a nice cleanup. Can we do that after landing this, though?

Reply:
I rewrote this method to no longer need MakeRequestClosure, which should void this...